src/types.js

```diff
@@ -85,3 +85,8 @@ export type Recipient = {
   subject: string,
   email: string,
 };
+
+export type ApiResponse = {
```

Comment:
I'd keep it for now - it'd just more code with any additional reason because responses for now looks almost the same.